packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -90,11 +90,7 @@ export const PaymentConfirmation = ({
         <header className="flex flex-col justify-center items-center row-divider-grey-200 text-center pb-8 mt-5 desktop:mt-2">
           {accountExists ? (
             <>
-              <img
-                className="max-h-12"
-                src={circledCheckbox}
-                alt="circled checkbox"
-              />
+              <img className="max-h-12" src={circledCheckbox} alt="" />
```

Comment:
What about L108-109 `alt=` tags?

https://github.com/mozilla/fxa/blob/91cb88066d891b0bc77cf391e333811dff1baec2/packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx#L108-L109

Reply:
Also, should all these have a specified img `width` and `height` to avoid potential CLS and stuff?

Reply:
Great catch! Created a follow up PR #16103